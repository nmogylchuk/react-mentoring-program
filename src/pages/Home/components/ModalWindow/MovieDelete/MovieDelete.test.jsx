import React from 'react';
import rerender from 'react-test-renderer';
import DeleteMovie from "./MovieDelete";
import {Provider} from 'react-redux'
import configureStore from 'redux-mock-store';
import {movie} from '../../../../../../__mocks__/mockData';

describe('when Delete Movie', () => {

    test('then snapshot delete', () => {
        const initialStore = {};
        const store = configureStore([])(initialStore);
        store.dispatch = jest.fn();
        const deleteMovieComponent = rerender.create(<Provider store={store}>
            <DeleteMovie onClose={jest.fn()} id={movie.id}/>
        </Provider>);
        const tree = deleteMovieComponent.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
