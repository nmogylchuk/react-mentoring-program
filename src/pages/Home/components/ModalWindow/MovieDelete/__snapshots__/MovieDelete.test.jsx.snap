// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when Delete Movie then snapshot delete 1`] = `null`;
